common/curl-p/curl_p.c

```diff
@@ -45,9 +45,12 @@ void transform(Curl *const ctx) {
   Curl s;
   size_t round = 0;
   for (; round < ctx->type; ++round) {
-    memcpy(s.state, ctx->state, sizeof(ctx->state));
-    sbox(ctx, &s);
+    if (round % 2 == 0)
```

Comment:
You can apply the trick of bitwise operations as following:
```C
if (number & 1)
    number is odd
else
    number is even
```

So that, rewrite to avoid heavy `%` operator as following:
```C
if (round & 1)
    sbox(ctx, &s);
else
    sbox(&s, cyx);
```

Reply:
Will do, thanks.